zerver/management/commands/realm_alias.py

```diff
@@ -32,19 +42,30 @@ def handle(self, *args, **options):
         realm = get_realm(options["string_id"])
         if options["op"] == "show":
             print("Aliases for %s:" % (realm.domain,))
-            for alias in do_get_realm_aliases(realm):
-                print(alias["domain"])
+            for alias in get_realm_aliases(realm):
+                if alias["subdomains_allowed"]:
+                    print("*." + alias["domain"])
+                else:
+                    print(alias["domain"])
             sys.exit(0)
 
-        domain = options['alias'].lower()
+        domain = options['alias'].strip().lower()
         if options["op"] == "add":
-            if not can_add_alias(domain):
+            try:
+                validate_domain(domain)
+                RealmAlias.objects.create(realm=realm, domain=domain, subdomains_allowed=options["subdomains_allowed"])
+                sys.exit(0)
+            except ValidationError as e:
+                print(e.messages[0])
+                sys.exit(1)
+            except DuplicateRealmAliasException:
                 print("A Realm already exists for this domain, cannot add it as an alias for another realm!")
                 sys.exit(1)
-            RealmAlias.objects.create(realm=realm, domain=domain)
-            sys.exit(0)
+            except IntegrityError:
+                print("This domain has already been added!")
+                sys.exit(1)
         elif options["op"] == "remove":
-            RealmAlias.objects.get(realm=realm, domain=domain).delete()
```

Comment:
For the management command, we can just print
`"Aliases for realm1: example.com (subdomains allowed), acme.com (subdomains not allowed), ..`

I think we want to avoid users having to know/understand the *. notation, especially since it is not quite accurate.